{"name": "test_login_notfounduser", "status": "passed", "attachments": [{"name": "stdout", "source": "6b648f0e-c441-487b-be27-8fe15cf09021-attachment.txt", "type": "text/plain"}], "start": 1749667848463, "stop": 1749667848566, "uuid": "5dee06b1-018c-4af9-ba6b-7e2c8e7bc94b", "historyId": "857ff9e2b048d3a4ec2d51d8ea12cb3d", "testCaseId": "857ff9e2b048d3a4ec2d51d8ea12cb3d", "fullName": "tests.views.test_auth_views#test_login_notfounduser", "labels": [{"name": "parentSuite", "value": "tests.views"}, {"name": "suite", "value": "test_auth_views"}, {"name": "host", "value": "PC-5CG4282G66"}, {"name": "thread", "value": "20736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.views.test_auth_views"}]}